applications/openshift/ocp_idp_no_htpasswd/rule.yml

```diff
@@ -14,3 +14,13 @@ severity: medium
 warnings:
     - general: |-
         {{{ openshift_cluster_setting("/apis/config.openshift.io/v1/oauths/cluster") | indent(8) }}}
+
+template:
+    name: yamlfile_value
+    vars:
+        file: cluster.yaml
+        yamlpath: ".spec.identityProviders[:].type"
```

Comment:
Just a guess. `rhel6` and `rhel7` CI nodes use `python2` to build the content. These `[:]` might be the problem when it's trying to load the `yaml` file.

This is the exact file that is causing the exception:
`Exception while handling file: /home/jenkins/workspace/scap-security-guide-pull-requests/label/rhel7/build/ocp4/rules/ocp_idp_no_htpasswd.yml`

Reply:
maybe this is related: https://github.com/google/rekall/pull/323/files

Reply:
I found the issue. I'll point it out in the right place.